kubert/src/server.rs

```diff
@@ -298,7 +300,7 @@ impl TlsCertPath {
     async fn load_certs(&self) -> std::io::Result<Vec<rustls::Certificate>> {
         // Open certificate file.
         let pem = tokio::fs::read(&self.0).await?;
-        let mut reader = std::io::BufReader::new(pem.as_slice());
+        let mut reader = BufReader::new(pem.as_slice());
```

Comment:
`BufRead` is [implemented for `&[u8]`](https://doc.rust-lang.org/nightly/std/io/trait.BufRead.html#impl-BufRead), so i don't _think_ the `BufReader` is necessary --- the whole thing is already buffered?
```suggestion
        let mut reader = pem.as_slice();
```